x-pack/test/cases_api_integration/security_and_spaces/tests/common/comments/patch_comment.ts

```diff
@@ -549,6 +551,48 @@ export default ({ getService }: FtrProviderContext): void => {
       }
     });
 
+    describe('partial updates', () => {
+      it('should not result to a version conflict (409) when adding a comment to an updated case', async () => {
```

Comment:
this should be 
```suggestion
      it('should not result to a version conflict (409) when updating a comment in an updated case', async () => {
```
right?

Reply:
Correct!